src/main/java/org/takes/Response.java

```diff
@@ -43,6 +43,11 @@
  *   "Content-Type", "text/html"
  * );</pre>
  *
+ * <p>The implementations of this interface may require that
```

Comment:
@bdragan are there any implementations with this requirement? If so, we should document them accordingly.